code/game/machinery/computer/shuttle.dm

```diff
@@ -61,6 +61,8 @@
 
 /obj/machinery/computer/shuttle/emag(mob/user as mob)
 	if(!emagged)

```

Comment:
I have no plans to replace every single "if(!emagged)" in the code with "if(machine_flags & EMAGGABLE)" , not in this PR at least.